server/@types/express/index.d.ts

```diff
@@ -12,12 +12,13 @@ declare module 'express-session' {
 
     booker: Booker
 
-    bookingsFuture?: OrchestrationVisitDto[]
-    bookingsPast?: OrchestrationVisitDto[]
-    bookingsCancelled?: OrchestrationVisitDto[]
-
     bookingJourney?: BookingJourney
     bookingConfirmed?: BookingConfirmed
+
+    bookings?: {
```

Comment:
Refactor how temporary cache of bookings is stored in session so typed `type` can be leveraged to remove duplication in controllers.